src/generateReport.js

```diff
@@ -46,14 +46,29 @@ const createReportData = config => {
 };
 const createRemoteReportData = (url, diffs) =>
   diffs.map(diff => {
-    const [browser, key, scenario] = diff.Key.split('/'); //eslint-disable-line no-unused-vars
-    const scenarioName = scenario.split('.png')[0];
+    const params = {};
+    if (diff.Key.split('/').length === 3) {
```

Comment:
A more optimal solution would be to store the 'split' result into a variable. At the moment the same split operation is being performed twice on the string.